msteams-platform/apps-in-teams-meetings/meeting-apps-apis.md

```diff
@@ -282,6 +284,96 @@ The following table includes the response codes:
 | **403** | The app is unable to send the signal. 403 response code can occur because of various reasons, such as the tenant admin disables and blocks the app during live site migration. In this case, the payload contains a detailed error message. |
 | **404** | The meeting chat doesn't exist. |
 
+## Targeted in-meeting notification API
```

Comment:
When we are referring to the API itself, let's call it 'Targeted Meeting Notification API'. Please update all other places which refer to the API itself, not the feature 🙏

This targeted meeting notification API can be used to send targeted in-meeting notification or a different notification feature in the future.